src/alpha-lab/phone-neon.md

```diff
@@ -57,15 +48,11 @@ Two visualisations that help to illustrate these patterns are heatmaps and scanp
 </div>
 
 ### Calculation of gaze metrics on AOIs
+Examining eye tracking data provides valuable insights into user behaviour, but relying solely on visualisations like heatmaps and scanpaths might not always reveal the complete picture. To gain a more comprehensive understanding, we conducted a quantitative analysis of our data. This involved calculating gaze metrics such as [dwell time](/alpha-lab/gaze-metrics-in-aois/#dwell-time) and [time to first contact](/alpha-lab/gaze-metrics-in-aois/#time-to-first-contact). These metrics offer tangible and quantitative outcomes about the salience of each AOI. 
 
-Analysing eye tracking data can provide valuable insights into user behaviour, but simply looking at visualisations like 
-heatmaps and scanpaths may not always reveal the full story. This is why we opted for a quantitative analysis of our data 
-as well by calculating gaze metrics, such as [dwell time](/alpha-lab/gaze-metrics-in-aois/#dwell-time) and
-[time to first contact](/alpha-lab/gaze-metrics-in-aois/#time-to-first-contact). These metrics offer tangible and 
-quantitative outcomes about the salience of each AOI: Longer dwell time implies longer total fixation on a specific AOI 
-and could be considered as a proxy of attentional allocation. Conversely, the shorter the time to first contact, the faster 
-this AOI captured the user's attention, pointing to increased salience of this area. Follow along with the AOI tutorial
-for these calculations and charts!
+Dwell time signifies the total duration of fixation on a specific AOI, serving as an indicator of attentional allocation. Longer dwell times suggest prolonged focus on an AOI. On the other hand, the time to first contact measures how quickly an AOI captures a user's attention, indicating its prominence.
```

Comment:
We could also add a short discussion of what our results show here. It can be brief, but would round the section off nicely